Makefile

```diff
@@ -216,7 +216,7 @@ test-devmode: test-prereq
 	rm -rf /tmp/getter*; rm -rf /tmp/trivy*
 	go test -failfast -tags dev -trimpath -race -cover -coverpkg ./... -coverprofile=coverage-dev-minimal.txt -covermode=atomic ./pkg/test/... ./pkg/storage/... ./pkg/extensions/sync/... -run ^TestInject
 	rm -rf /tmp/getter*; rm -rf /tmp/trivy*
-	go test -failfast -tags stress,$(BUILD_LABELS) -trimpath -race -timeout 15m ./pkg/cli/server/stress_test.go
+	go test -failfast -tags stress,$(BUILD_LABELS) -trimpath -race -timeout 15m ./pkg/cli/server/stress_test.go ./pkg/cli/server/rlimit_linux_test.go
```

Comment:
why is this needed?

Reply:
I understand, and thank for picking it up.
The part I was not clear about is this change specifically. Doesn't ./pkg/cli/server/stress_test.go import the new type without this line change? ./pkg/cli/server/rlimit_linux_test.go shouldn't even be specified since it is specific to a hardcoded platform value.

Reply:
Moving the type declarations to `zotregistry.dev/zot/pkg/test/common` did avoid the Makefile change and seems to be working so far.